index.html

```diff
@@ -52,25 +52,25 @@
           </div>
         </div>
         <div class="menu-item__size">
-          <span class="select" title="字体">小四</span>
+          <span class="select" title="Font Size">Small Four</span>
           <div class="options">
             <ul>
```

Comment:
Let's not update index.html since it is not imported in our application

Reply:
Okay.